portfolio/src/main/webapp/script.js

```diff
@@ -131,3 +139,13 @@ function addLogInOutButton(url, isLoggedIn) {
   }
   
 }
+
+function showHideElement(container, hide) {
```

Comment:
nit: this is a strange method name. If the input variable is `hide` maybe the method should be `shouldHideElement(container, hide)`